test/utils/run_tests.sh

```diff
@@ -22,30 +27,47 @@ runTest ( ) {
         rm -f test.db.sqlite3
     fi
 
-    if [ "$2" = "test" ]; then
-        "${mocha}"
-    elif [ "$2" = "coverage" ]; then
-        "${nyc}" --reporter=lcov node_modules/.bin/_mocha
+    if [ "x$3" = "x" ] || [ "$3" = "single_process" ]; then
```

Comment:
For future compatibility with WMF CI runs, it would be good for both `$2` and `$3` to be able to be set via `TEST_TARGET` and `TEST_MODE` (and for env vars to be preferred over the positional args).

Reply:
I was thinking about this, and I think we need to just remove the test runner script. Fallback to SQLite is default now via the default value of the RB_TEST_BACKEND env variable in the config. Locally running all configurations is unlikely to happen a lot, in Travis we can achieve running all the configs via env variables matrix. I think I'll just document the env variables and remove the run_tests.sh. What do you think?

Reply:
Doing that.

Reply:
I actually liked your earlier idea about removing this altogether and control stuff only with ENV VARS.